frontend/src/__mocks__/mockSingleModelRegistry.ts

```diff
@@ -0,0 +1,143 @@
+export const mockSingleModelRegistry = {
+  apiVersion: 'modelregistry.opendatahub.io/v1alpha1',
+  kind: 'ModelRegistry',
+  metadata: {
+    annotations: {
+      'kubectl.kubernetes.io/last-applied-configuration':
+        '{"apiVersion":"modelregistry.opendatahub.io/v1alpha1","kind":"ModelRegistry","metadata":{"annotations":{},"labels":{"app.kubernetes.io/created-by":"model-registry-operator","app.kubernetes.io/instance":"modelregistry-sample","app.kubernetes.io/managed-by":"kustomize","app.kubernetes.io/name":"modelregistry","app.kubernetes.io/part-of":"model-registry-operator"},"name":"modelregistry-sample","namespace":"shared"},"spec":{"grpc":{"port":9090},"postgres":{"database":"model-registry","host":"model-registry-db","passwordSecret":{"key":"database-password","name":"model-registry-db"},"username":"mlmduser"},"rest":{"port":8080,"serviceRoute":"disabled"}}}\n',
+    },
+    creationTimestamp: '2024-03-14T08:01:42Z',
+    finalizers: ['modelregistry.opendatahub.io/finalizer'],
+    generation: 1,
+    labels: {
+      'app.kubernetes.io/created-by': 'model-registry-operator',
+      'app.kubernetes.io/instance': 'modelregistry-sample',
+      'app.kubernetes.io/managed-by': 'kustomize',
+      'app.kubernetes.io/name': 'modelregistry',
+      'app.kubernetes.io/part-of': 'model-registry-operator',
+    },
+    managedFields: [
```

Comment:
can you remove managedField, and make it as an empty array, it's not necessary for mocks